### PR TITLE
👌 IMPROVE: Add Emoji Log to extensions.json

### PR DESCRIPTION
**Pull request by xcesiv**
https://github.com/ahmadawais/Emoji-Log

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -1343,6 +1343,10 @@
     {
       "id": "shyykoserhiy.git-autoconfig",
       "repository": "https://github.com/ShyykoSerhiy/git-autoconfig"
+    },
+    {
+      "id": "ahmadawais.emoji-log",
+      "repository": "https://github.com/ahmadawais/Emoji-Log"
```
Comment (jankeromnes) [inline extensions.json:1349]: I see that this repository publishes official release tags.

When that's the case, we prefer to pin a specific version & publishing from the corresponding release tag, as opposed to just publishing from the default branch (may contain WIP changes).

```suggestion
      "repository": "https://github.com/ahmadawais/Emoji-Log",
      "version": "0.7.0",
      "checkout": "v0.7.0"
```

**Review by jankeromnes — APPROVED** [review]
Excellent, many thanks!

One comment below, but I'll be able to apply the suggestion automatically on GitHub before merging, so this is good to go. 🚀

File: extensions.json (hunk 1 above)
Comment (xcesiv) [reply extensions.json]: Awesome! thank you.

**Comment by jankeromnes** [comment]
@xcesiv Oops, automated publishing failed with:

```
Running: git clone --recurse-submodules https://github.com/ahmadawais/Emoji-Log /tmp/repository
Running: git checkout v0.7.0
Running: npm install
[FAIL] Could not process extension: {
  "id": "ahmadawais.emoji-log",
  "repository": "https://github.com/ahmadawais/Emoji-Log",
  "version": "0.7.0",
  "checkout": "v0.7.0"
}
Error: Missing publisher name. Learn more: https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions
```

I guess that's because their `package.json` is missing a `"publisher": "ahmadawais"` entry?

**Comment by xcesiv** [comment]
@jankeromnes I addressed the issue in #233 looks like he has a separate repository - [Emoji-Log-VSCode](https://github.com/ahmadawais/Emoji-Log) dedicated to it that I missed. I made sure to include the [current version](https://github.com/ahmadawais/Emoji-Log-VSCode/releases/tag/1.0.0). 

The `"publisher": "ahmadawais"` entry is found here:

https://github.com/ahmadawais/Emoji-Log-VSCode/releases/tag/1.0.0

Let me know if there is anything else necessary.

**Pull request merged**